src/App.js

```diff
@@ -1,38 +1,33 @@
-import React, { useState } from "react";
-import "./App.css";
+import React, { useState } from 'react';
+import './App.css';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
   Link,
   useRouteMatch,
-} from "react-router-dom";
+} from 'react-router-dom';
 
-import NotFound from "./components/NotFound";
-import AddItem from "./components/AddItem";
-import ListView from "./components/ListView";
-import Home from "./components/Home";
+import NotFound from './components/NotFound';
+import AddItem from './components/AddItem';
+import ListContainer from './components/ShoppingList/ListContainer';
+import Home from './components/Home';
 
 const App = () => {
-  const [auth, setAuth] = useState(localStorage.getItem("token"));
+  const [auth, setAuth] = useState(localStorage.getItem('token'));
 
   return (
     <Router>
       {auth ? (
         <div className="App">
-          <MenuLink
-            activeWhenExact={true}
-            pathTo="/ListView"
-            label="List View"
-          />
+          <MenuLink activeWhenExact={true} pathTo="/List" label="List View" />
           <MenuLink pathTo="/AddItem" label="Add Item" />
-
           <Switch>
             <Route exact path="/">
-              <ListView />
+              <ListContainer />
             </Route>
-            <Route exact path="/ListView">
-              <ListView />
+            <Route exact path="/List">
+              <ListContainer />
             </Route>
```

Comment:
I noticed that the exact path changed from "/ListView" to "/List". In some parts of the code there is `history.push('/ListView');` and others there are `history.push('/List')`

I think we need to stick to one or the other. Right now when I clear the token in LocalStorage and then I click "add item" on home screen, it takes me to `/ListView` but I get a 404 - Not Found! page. Keeping the path consistent throughout the codebase might resolve this bug. :smile:

Reply:
Good call Maeesha! I'm having the same issues but I think this should fix them.

Reply:
Definitely agree! I thought we snagged all old necessary instances of ListView with routing. The idea was to help differentiate between the parent 'Container' or list status so we can generally always run that conditional while preserving the existing 'ListView' component. Maybe we rename 'ListContainer' to 'ListView' and current 'ListView' to 'FullList' for better parity within that cluster?

Reply:
I think the way you guys did it makes sense - I just commented on the two places in the "Home" component that weren't changed which is where the issue is happening